scalafix-core/shared/src/main/scala/scalafix/rule/Rule.scala

```diff
@@ -124,7 +128,9 @@ abstract class SemanticRule(index: SemanticdbIndex, name: RuleName)
 
 object Rule {
   private[scalafix] class CompositeRule(val rules: List[Rule])
-      extends Rule(rules.foldLeft(RuleName.empty)(_ + _.name)) {
+      extends Rule(
+        rules.foldLeft(RuleName.empty)(_ + _.name),
+        rules.foldLeft("")(_ + _.description)) {
```

Comment:
Hmm, I think we may want to add something like `private[scalafix] def metadata: List[RuleMetadata]` where 
```scala
package scalafix.internal.rule
case class RuleMetadata(name: RuleName, description: String)
```